workshop/exercise-4/README.md

```diff
@@ -1,108 +1,96 @@
-## Clone the Application Repo and Deploy our Application using Kubectl and yaml
+## Deploy vnext Version and Apply Traffic Shifting
 
-### Clone the application repo
-Let's get the code we'll use for today's lab. This repository contains the code for the Fibonacci application as well as various .yaml files we'll use throughout the lab.
+Did you notice that the Fibonacci sequence started with 1? Many would argue that the sequence should actually start with 0. There's a vnext version of the application that starts the sequence with 0 instead of 1. This container image has been built and deployed to dockerhub, and tagged as vnext. We'll deploy that as v2 of our app, and then route a small percentage of the traffic to it.
 
-1. Clone the git repository:
+![](https://github.com/knative/serving/raw/master/docs/spec/images/fibknativev2.png)
+
+### Update First Revision Name
+1. When we first deployed our application, we didn't provide a revision name, so Knative assigned a random revision name, something like `fib-knative-lhghx-1`. Let's give the service the revision name `fib-knative-one` since in this version of the application the sequence begins with one. Naming our own revisions can be helpful for readability, but isn't required. We'll later use this revision name to route traffic between two different revisions.
```

Comment:
thanks! I like it much better to keep the first `kn` usage as trivial as possible!